WordPress/Classes/ViewRelated/Tools/Time Zone/Views/TimeZoneTableViewCell.swift

```diff
@@ -0,0 +1,132 @@
+        label.font = UIFont(descriptor: fontDescriptor, size: fontDescriptor.pointSize)
+        label.textColor = .secondaryLabel
+
+        if label.effectiveUserInterfaceLayoutDirection == .leftToRight {
+            // swiftlint:disable:next natural_text_alignment
+            label.textAlignment = .left
+        } else {
+            // swiftlint:disable:next inverse_text_alignment
+            label.textAlignment = .right
+        }
+
+        return label
+    }()
+
+    lazy var rightSubtitle: UILabel = {
+        let label = UILabel()
+        setupTimeZoneLabel(label)
+        let fontDescriptor = UIFontDescriptor.preferredFontDescriptor(withTextStyle: .caption1)
+        label.font = UIFont(descriptor: fontDescriptor, size: fontDescriptor.pointSize)
+        label.textColor = .secondaryLabel
+
+        if label.effectiveUserInterfaceLayoutDirection == .leftToRight {
+            // swiftlint:disable:next inverse_text_alignment
+            label.textAlignment = .right
+        } else {
+            // swiftlint:disable:next natural_text_alignment
+            label.textAlignment = .left
+        }
+
+        return label
+    }()
+
+    // MARK: - Initializers
+
+    public required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        setupSubviews()
+    }
+
+    public required override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        setupSubviews()
+    }
+
+    func setupTimeZoneLabel(_ label: UILabel) {
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.numberOfLines = 1
+        label.adjustsFontForContentSizeCategory = true
+    }
+
+    private func setupSubviews() {
+        // Not every WPTimeZone has a time zone offset so wrapping content in UIStackView
+        // to allow for dynamic resizing for these cases
+        let stackView = UIStackView()
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.axis = .vertical
+        stackView.alignment = .fill
+
+        let subtitleContainerView = UIView()
+        subtitleContainerView.translatesAutoresizingMaskIntoConstraints = false
+        subtitleContainerView.addSubview(leftSubtitle)
+        subtitleContainerView.addSubview(rightSubtitle)
+
+        stackView.addArrangedSubviews([titleLabel, subtitleContainerView])
+
+        configureConstraints(subtitleContainerView: subtitleContainerView)
+
+        contentView.addSubview(stackView)
+        contentView.pinSubviewToAllEdges(stackView, insets: UIEdgeInsets(
+                top: Constants.topPadding,
+                left: Constants.leadingTrailingPadding,
+                bottom: Constants.bottomPadding,
+                right: Constants.leadingTrailingPadding)
+        )
+    }
+
+    private func configureConstraints(subtitleContainerView: UIView) {
```

Comment:
The constraints here could be simplified by using a stackview for the subtitle labels. In fact, we wouldn't need this method at all 🙂